grammar.js

```diff
@@ -514,12 +516,21 @@ module.exports = grammar({
 
     create_table: $ => seq(
       $.keyword_create,
-      optional($._temporary),
+      optional(
+          choice(
+              $._temporary,
+              $.keyword_unlogged
+          )
+      ),
       $.keyword_table,
       optional($._if_not_exists),
       $.table_reference,
-      $.column_definitions,
+      optional($.column_definitions),
       optional($.table_options),
+      optional(seq(
+          $.keyword_as,
+          $._select_statement,
```

Comment:
this should be a choice() between column_definitions and the `as select` clause rather than allowing both. The table_options I think follow column_definitions for MySQL if used so that'd be another seq, but doublecheck me on that one!

Reply:
Thanks. I have fixed that. I am planning to put a lot of the Hive specific statements (for #69) in table options.

I am not 100% sure about the MySQL reference, because the create_definition is wrapped in [] and ()...
I think I will test this out.

Reference: https://dev.mysql.com/doc/refman/8.0/en/create-table.html
```sql
CREATE [TEMPORARY] TABLE [IF NOT EXISTS] tbl_name
    [(create_definition,...)]
    [table_options]
    [partition_options]
    [IGNORE | REPLACE]
    [AS] query_expression
```

Reply:
This is a valid query in MySQL (but not in Postgres, even if MAX_ROWS is not there)

```sql
create table books2 
(a int)
MAX_ROWS=100
as select * from books
```

Reply:
oh interesting! I think we still need a `choice` because you _have_ to specify something, you can't just end it at `create table name` without specifying anything, but it'd be more like this:

```js
      ...,
      $.table_reference,
      choice(
          seq(
              $.column_definitions,
              optional($.table_options),
              optional(seq(
                $.keyword_as,
                $._select_statement,
              )),
          ),
          seq(
              optional($.table_options),
              seq(
                $.keyword_as,
                $._select_statement,
              ),
          ),
      )
```
so down the first branch you have to have the column definition and down the other you have to have `as select`.

Reply:
Oh, yeah. I had too many optional() in the last version.

One other thing, that is currently wrong is the `table_option` in line 855.
I have put the `ON COMMIT` in there. Should I create a new named node for that? There many more of these kinds of statements, that are optional, specified once after the "main" statement" and in a particular order. This gets really complicated, even when only considering only one dialect.

Also, `ON COMMIT` takes only effect when the table is `TEMP` or `TEMPORARY`.

Reply:
when to break out a named node is one of those more-art-than-science things but the two main reasons I ever have to do it are, one, it's reused elsewhere, and two, it represents a certain fuzzy level of internal complexity. The first doesn't apply here afaik, but I think you could make a good case for the second if you're inclined to do that.

Are you sure the options are ordered? I skimmed the 8.0 docs and they present table options alphabetically without an obvious mention of ordering.

When `on commit` does something is a distinct question from whether it parses -- as long as it's a no-op rather than a syntax error for permanent relations we're all good there. If it _is_ a syntax error in that case then I'm still not sure it's worth the effort of splitting temporary and permanent versions of `create table`.

Reply:
I have removed `on commit` for now, but kept the keywords. 
Regarding the other options mentioned above, I will include the partially when working on the hive specific commands for #69.